openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml

```diff
@@ -32,9 +32,31 @@
   implementation_status: planned
   narrative:
     - text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        Configuration management is enforced through Ansible playbooks,
+        Ignition configurations and Operators.
+
+        Ansible features a state-driven resource model that describes
+        the desired state of computer systems and services, allowing
+        reliable and repeatable IT infrastructure configuration. The
+        configuration is documented on the the system [Repository] site.
+
+        Ignition is the utility that is used by Red Hat Enterprise Linux CoreOS(RHCOS)
+        to manipulate disks during initial configuration, confirming all new machines
+        meet the declared configuration.
```

Comment:
We should probably add this write up for Ignition and MCO to the CoreOS component no?